code42.json

```diff
@@ -3150,102 +3295,18 @@
                     "data_type": "numeric"
-                },
-                {
-                    "data_path": "action_result.parameter.start_date",
-                    "data_type": "string"
-                },
-                {
-                    "data_path": "action_result.parameter.end_date",
-                    "data_type": "string"
-                },
-                {
-                    "data_path": "action_result.parameter.file_hash",
-                    "data_type": "string",
-                    "contains": [
-                        "md5",
-                        "sha256"
-                    ]
-                },
-                {
-                    "data_path": "action_result.parameter.file_path",
-                    "data_type": "string",
-                    "contains": [
-                        "file path"
-                    ]
-                },
-                {
-                    "data_path": "action_result.parameter.file_category",
-                    "data_type": "string"
-                },
-                {
-                    "data_path": "action_result.parameter.username",
-                    "data_type": "string",
-                    "contains": [
-                        "email",
-                        "user name"
-                    ]
-                },
-                {
-                    "data_path": "action_result.parameter.hostname",
-                    "data_type": "string",
-                    "contains": [
-                        "host name"
-                    ]
-                },
-                {
-                    "data_path": "action_result.parameter.private_ip",
-                    "data_type": "string",
-                    "contains": [
-                        "ip",
-                        "ipv6"
-                    ]
-                },
-                {
-                    "data_path": "action_result.parameter.public_ip",
-                    "data_type": "string",
-                    "contains": [
-                        "ip",
-                        "ipv6"
-                    ]
-                },
-                {
-                    "data_path": "action_result.parameter.exposure_type",
-                    "data_type": "string"
-                },
-                {
-                    "data_path": "action_result.parameter.process_name",
-                    "data_type": "string"
-                },
-                {
-                    "data_path": "action_result.parameter.url",
-                    "data_type": "string",
-                    "contains": [
-                        "url"
-                    ]
-                },
-                {
-                    "data_path": "action_result.parameter.window_title",
-                    "data_type": "string"
                 }
             ],
             "render": {
-                "type": "table"
+                "type": "custom",
+                "view": "code42_view.display_view"
```

Comment:
This is what tells the app to use the custom view method from `code42_view.py` `display_view()`